server/db/index.js

```diff
@@ -1,2 +1,10 @@
 //Everything comes together here (models from hooks)
 //export db, models
+
+const db = require('./db')
+const Order = require('./models/Order')
+const Product = require('./models/Product')
+const ProductOrders = require('./models/ProductOrders')
+
```

Comment:
These models should be imported from hooks.js (even if you dont have any hooks set up yet)